service/service.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/giantswarm/app-operator/v2/pkg/project"
 	"github.com/giantswarm/app-operator/v2/service/controller/app"
 	"github.com/giantswarm/app-operator/v2/service/controller/appcatalog"
-	"github.com/giantswarm/app-operator/v2/service/watcher/configmap"
+	"github.com/giantswarm/app-operator/v2/service/watcher"
```

Comment:
Since we are supporting secret watching on the same package, no more need to name it `configmap`.